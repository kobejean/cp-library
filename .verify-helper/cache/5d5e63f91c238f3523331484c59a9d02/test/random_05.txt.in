97
151732181 27813731 76291170
919805912 294388033 956595199
941792897 161141668 354780394
622798641 15759678 109622302
548446667 435214895 696971770
882542877 150414847 678609960
244737823 211920049 601824619
567404334 222772098 759902844
88232428 87401230 676785192
532622288 477971406 497372821
244039406 7256728 183483100
540799439 14551161 397736274
813267166 429496194 688990830
637343026 517720012 215348446
704648771 550284387 786363550
491176438 120903884 18727152
989199801 535576491 56473196
596114934 409790742 246281228
842383458 186780260 504022108
183875885 138342950 127195355
697465856 326902606 749716808
334057216 258882844 274763210
169399301 115557888 742698704
364151042 311146272 792617484
368779572 35565530 41293742
442040953 6138552 120636345
221776893 6523190 408923949
111560623 10239497 667333736
92549986 70922286 845078788
552711910 309029010 208692156
722090137 35276693 795572679
830683196 525470239 989500029
7936576 655670 753888540
201445127 1323413 837517980
488341031 242791909 802208903
30021161 6721474 328203984
368830466 261720504 826786496
946153134 735947907 645794349
871643153 629550821 616466006
286439679 215365539 575969538
652104260 602346816 726486848
260219083 215285878 780657249
16234580 1672232 194814960
925198969 220323657 925198969
461047443 440406445 922094886
574202093 381180955 574202093
760615656 445015672 760615656
904974283 869039391 904974283
375055338 29954972 375055338
257662039 78455546 257662039
611655435 222713173 611655435
185084161 102332823 925420805
45101602 17931623 856930438
830800147 493848031 830800147
282513246 210416132 565026492
892522811 31418149 892522811
303553264 83892440 910659792
182360027 138537844 182360027
743444196 714293132 743444196
999971629 168746689 999971629
238350231 43195918 476700462
64668559 23797333 388011354
998715849 287640342 998715849
999999990 199999998 433333329
999999976 14705882 926470566
999999972 916666641 638888871
999999993 470588232 705882348
1000000000 100000000 700000000
999999994 571428568 428571426
999999965 234042545 425531900
999999963 298245603 491228052
999999984 979166651 979166651
999999975 199999995 107692305
999999980 257142852 685714272
999999996 583333331 916666663
999999995 662921345 134831460
999999984 124999998 708333322
999999990 212121210 893939385
999999993 725490191 78431372
999999982 517241370 482758612
999999952 214285704 982142810
999999982 105263156 710526303
999999990 954545445 560606055
999999972 222222216 694444425
1000000000 850000000 650000000
999999999 222222222 222222222
999999990 222222220 20202020
1000000000 500000000 250000000
999999990 555555550 722222215
999999980 467391295 771739115
999999996 333333332 833333330
1000000000 800000000 800000000
999999959 69767439 46511626
999999976 794117628 264705876
1000000000 300000000 900000000
999999999 666666666 333333333
999999924 440476157 130952371
