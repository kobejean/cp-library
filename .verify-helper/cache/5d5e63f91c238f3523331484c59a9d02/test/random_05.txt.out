22151367
17
941792870
604959783
82
-1
70
94567379
-1
74
122019687
-1
10
318671481
-1
46
31909644
217952592
13
36775147
-1
10
7365154
88280190
17
7246496
-1
60
46274949
264929650
29
118669009
-1
19
28725922
-1
55
315384371
-1
23
163025978
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
18
41
33
5
7
1
30
34
47
26
4
7
47
21
2
29
1
12
28
9
4
11
8
88
2
2
59
2
4
20
31
3
1
73
